python/mxnet/numpy/multiarray.py

```diff
@@ -44,7 +44,7 @@
 from ..ndarray.numpy import _internal as _npi
 
 __all__ = ['ndarray', 'empty', 'array', 'zeros', 'ones', 'add', 'subtract', 'multiply', 'divide',
-           'mod', 'power']
+           'mod', 'power', 'vstack']
```

Comment:
after rebase, put `vstack` to after `stack` in this list and within the file, same rule applies to the other 2 files too.

Reply:
Done.